webhook_handlers/views/github.py

```diff
@@ -505,7 +505,11 @@ def _handle_marketplace_events(self, request, *args, **kwargs):
             if subscription.status == "active":
                 log.warning(
                     "GHM webhook - user purchasing but has a Stripe Subscription",
-                    extra=dict(username=username),
+                    extra=dict(
+                        username=username,
+                        plan_name=subscription.plan.get("name", None),
```

Comment:
Should we call this `old_plan_name` to be more explicit?

Reply:
Sounds good! Will do